crates/sui-node/src/metrics.rs

```diff
@@ -119,21 +124,28 @@ pub fn start_metrics_push_task(config: &sui_config::NodeConfig, registry: Regist
         url: &reqwest::Url,
         registry: &RegistryService,
     ) -> Result<(), anyhow::Error> {
-        let metrics = TextEncoder
-            .encode_to_string(&registry.gather_all())
-            .context("encoding metrics")?;
+        let mut buf: Vec<u8> = vec![];
+        let encoder = prometheus::ProtobufEncoder::new();
+        encoder.encode(&registry.gather_all(), &mut buf)?;
 
         let response = client
             .client()
             .post(url.to_owned())
-            .body(metrics)
+            .header(header::CONTENT_TYPE, "application/mysten.proxy")
```

Comment:
it does, but this is the content-type sent from the sui-node to the sui-proxy.  So i just kinda made it up.  Technically i suppose it could be a similar header but it was pretty complicated and we really only use it to inspect/check against a value. so it doesn't ultimately matter between our node and our proxy.